### ISSUE TITLE
Vogt’s Striae

### ISSUE THREAD

**Issue by daboe01** [obophenotype/human-phenotype-ontology#10901]
Preferred term label:
Vogt’s Striae

Synonyms
Vertical stromal lines, Corneal striae

Definition (free text, please give PubMed ID)
Vertical lines in the posterior stroma of the cornea, aligned with Descemet’s membrane. These lines temporarily disappear with pressure applied to the cornea. Vogt’s striae are a hallmark feature of keratoconus but have also been observed in healthy eyes. Rare cases of horizontal striae have been reported.

Parent term (use hpo.jax.org/app)
Abnormality of the corneal stroma (HP:11492)

Diseases characterized by this term
Keratoconus (HP:563)

Your nano-attribution (ORCID)
0000-0001-9042-0508

**Comment by pnrobinson** [comment]
@daboe01 
Adding new term: 
- ID: HP:6001180
- label: Vogt striae
- parents: [Abnormal corneal stroma morphology (HP:0011492)](https://hpo.jax.org/app/browse/term/HP:0011492)
- pmid: PMID:18700919
- def: Vertical lines in the posterior stroma of the cornea, aligned with Descemet membrane. These lines temporarily disappear with pressure applied to the cornea. Vogt striae are a hallmark feature of keratoconus but have also been observed in healthy eyes. Rare cases of horizontal striae have been reported.
- comment: 
- orcid: orcid:0000-0001-9042-0508
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10901
- synonyms: Vertical stromal lines(NONE)|Corneal striae(NONE)|Vogt’s striae(NONE)
